src/Data/Product/Properties/WithK.agda

```diff
@@ -8,33 +8,18 @@
 
 module Data.Product.Properties.WithK where
 
-open import Data.Bool.Base
 open import Data.Product
-open import Data.Product.Properties using (,-injectiveˡ)
 open import Function
-open import Relation.Binary using (Decidable)
 open import Relation.Binary.PropositionalEquality
-open import Relation.Nullary.Reflects
-open import Relation.Nullary using (Dec; _because_; yes; no)
-open import Relation.Nullary.Decidable using (map′)
+open import Data.Product.Properties as P using ()
 
 ------------------------------------------------------------------------
 -- Equality
 
-module _ {a b} {A : Set a} {B : Set b} where
-
-  ,-injective : ∀ {a c : A} {b d : B} → (a , b) ≡ (c , d) → a ≡ c × b ≡ d
-  ,-injective refl = refl , refl
+,-injective = P.,-injective
+≡-dec = P.≡-dec
```

Comment:
Hmm don't quote me on this, but I think this will still clash if you import both modules? Even if it doesn't the more idiomatic way to do this is:
```
open import Data.Product.Properties public
  using (,-injective, ≡-dec)
```